### DISCUSSION TITLE
webhook configure with ory kratos

### DISCUSSION THREAD

**Discussion by nihalgurjar47** [ory/kratos#3725] [Q&A]
Hii @hperl  @radekg ,
how to webhook configure in ory kratos?

My requirement is to check the metadata_admin details before logging in. If metadata_admin is null, an error will occur, redirecting to another page; otherwise, the response will be 200, allowing the user to log in successfully.

Note: I have follow the documentation but did not get result.

please share the reference.

**Comment by MJEND7** [comment]
Hey @nihalgurjar47 I have had a similar issue, with out seeing your config for web hooks it's hard for me to help. But for me I needed to 

1. Have the hook run is the right place depending on the auth type I was working with.
2. Make sure I was calling the right method and sending the correct body.

Also what kind of webhook are you trying to send? Are you just trying to ping your api?

Very important to note that if you want the auth to fail out if the webhook fails to include the `ignore` field.
```yml
  response:
    ignore: false # Defines if the webhook response should be ignored and run async. Boolean. OPTIONAL
```
